apps/functions-subscription/src/adapters/azure/functions/__tests__/data.ts

```diff
@@ -123,3 +131,15 @@ export const makeAValidGetTrialRequest = () =>
       trialId: aTrialId,
     },
   });
+
+export const makeAValidListTrialRequest = (params?: {
+  readonly pageSize?: string;
+  readonly maximumId?: string;
+  readonly minimumId?: string;
+}) =>
```

Comment:
we do not have a type that includes all three parameters, nor a type generated from a schema (we declared the schema in the parameters themselves).

we could use the `ItemId` type for `maximumId` and `minimumId`, but since we're mocking the params sent with the request we can only use string or string-like types (`NumberFromString` in `pageSize` would not work, or would require to stringify the value).

a better solution i think would be to create a dto with this same structure, if necessary